### PR TITLE
Feature project update view

### LINKED ISSUE #50: Manager Base Interface
- [ ]  Dashboard with available projects
- [ ]  Interface with people assigned to projects (feature moved to creating raport feature)
- [ ]  Interface with all details of chosen project

Manager Base Interface is directed to the Admin-type and Manager-type user
- for Admin-type user:
- [x] **List** with available projects
- [x] Interface with all **details** of chosen project
- [x] Interface for **updating** details of chosen project
- [ ] Interface for **creating** new project
- [ ] Interface for **deleting** project

- for Manager-type user:
- [x] **List** with manager's projects
- [x] Interface with all **details** of manager's project
- [x] Interface for **updating** details of manager's project
- [ ] Interface for **deleting** manager's project

### LINKED ISSUE #51: Create tests for projects views
Create necessary tests for managers app views

- [ ] Tests for ProjectsList APIviews

- [ ] Tests for ProjectsDetail APIviews

- [ ] Tests for ProjectsUpdateAPIviews

- [ ] Tests for ProjectsCreate APIviews

- [ ] Tests for ProjectsDelete APIviews

### PR DESCRIPTION
**Pull request by MartynaAnnaGottschling**
Resolves #50
Resolves #51

### CODE REVIEW

File: managers/templates/managers/project_update.html [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  </small>
+  &nbsp;{{project.name}}&nbsp;
+</h2>
+
+<div class="modal-dialog" style="margin-bottom:0">
+  <form action="{% url 'custom-project-update' pk=project.id %}" method="post">
+      {% csrf_token %}
+      {% render_form serializer template_pack='rest_framework/vertical' %}
+          <div class="tool_tip">
+            <font color="grey">How to add more employees?</font>
```
Comment (rwrzesien) [inline managers/templates/managers/project_update.html]: Please add translations where needed.

File: managers/templates/managers/project_update.html [hunk 2]
```diff
@@ -0,0 +1,32 @@
+{% extends 'base.html' %}
+{% block content %}
+{% load i18n %}
+{% load rest_framework %}
+{% load static %}
+<head>
```
Comment (rwrzesien) [inline managers/templates/managers/project_update.html]: This should be put in `{% block extra_head %}` which should be defined in `base.html` at the end of `<head>` tag.

File: managers/tests/test_unit_managers_apiview.py [hunk 3]
```diff
@@ -37,6 +37,7 @@ def setUp(self):
         self.url = [
             reverse('custom-projects-list'),
             reverse('custom-project-detail', args=(self.project.pk,)),
+            reverse('custom-project-update', args=(self.project.pk,)),
```
Comment (rwrzesien) [inline managers/tests/test_unit_managers_apiview.py]: Same here, please overwrite `self.url` in `ProjectUpdateTests` `setUp` method.

File: managers/views.py [hunk 4]
```diff
@@ -44,3 +45,32 @@ class ProjectDetail(APIView):
     def get(self, request, pk):
         project = get_object_or_404(Project, pk=pk)
         return Response({'project': project})
+
+
+class ProjectUpdate(APIView):
```
Comment (rwrzesien) [inline managers/views.py]: Same here, lets discuss how this should actually work - as API, or as a classic django view with template.

File: managers/templates/managers/project_update.html (hunk 1 above)
Comment (MartynaAnnaGottschling) [reply managers/templates/managers/project_update.html]: Done

File: managers/tests/test_unit_managers_apiview.py (hunk 3 above)
Comment (MartynaAnnaGottschling) [reply managers/tests/test_unit_managers_apiview.py]: Done

File: managers/views.py (hunk 4 above)
Comment (MartynaAnnaGottschling) [reply managers/views.py]: It will be done in separate pull request

File: managers/templates/managers/project_update.html (hunk 2 above)
Comment (MartynaAnnaGottschling) [reply managers/templates/managers/project_update.html]: Done

File: managers/views.py (hunk 4 above)
Comment (rwrzesien) [reply managers/views.py]: :+1: done

**Comment by MartynaAnnaGottschling** [comment]
@kbeker 
All changes seem legit...but I'm so sad, that my tooltip was cut off :D It is good that we are getting rid of Django REST things :+1:

**Pull request merged**